controller/users.go

```diff
@@ -94,14 +94,14 @@ func (c *UsersController) Obfuscate(ctx *app.ObfuscateUsersContext) error {
 				"user_id": u,
 				"err":     err,
 			}, "unable to load the user")
-			return jsonapi.JSONErrorResponse(ctx, goa.ErrNotFound(err))
+			return err
 		}
 		if len(users) != 1 {
 			log.Error(ctx, map[string]interface{}{
 				"user_id": u,
 				"err":     err,
 			}, "unable to find the user")
-			return jsonapi.JSONErrorResponse(ctx, goa.ErrNotFound(errs.New("unable to find the user")))
+			return errors.NewNotFoundError("unable to find the user", u.String())
```

Comment:
This is the proper way because the message is added automatically: `return errors.NewNotFoundError("user", u.String())`